src/pages/en/guides/cms/storyblok.md

```diff
@@ -85,8 +85,8 @@ export default defineConfig({
         // Add your components here
       },
       apiOptions: {
-        // Choose your Storyblok space region 
-        region: 'us',
+        // Choose your Storyblok space region
+        region: 'us', // or 'eu'
```

Comment:
[optional] `// or 'eu' (default)`

Reply:
The region is mentioned in a callout before but it doesn't hurt to be repeated. I spent a good 2 hours debugging this small option when I first used Storyblok and that's the reason It is included here.

Reply:
Agreed. I think it's good to mention both because following the storybook docs, they only suggest adding 'us' if you need it (leave blank for eu).

If you're skimming the article it'd be very easy for an EU user to miss that this has been flipped, even with the which is why I added the comment to help draw attention to this area even if you're quickly copying and pasting.